scripts/display-travis-status.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const debug = require('debug')('display_travis_status')
+const pollTravis = require('../lib/pollTravis')
+
+module.exports = function (app) {
+  app.on('pull_request.opened', (event) => {
+    const owner = event.repository.owner.login
+    const repo = event.repository.name
+    if (repo !== 'nodejs.org') return
```

Comment:
Yup, I wanted to make sure this was focused on what it was written for when I started touching it. I figured that was _probably_ the case... but I'm not directly involved with that, so I leaned towards sheltering it from other repos to start.

I can add additional repo names now if you want... or it can come in another PR. LMK!